Data/market-hours/market-hours-database.json

```diff
@@ -338,6 +338,8 @@
         "7/3/2013": "13:00:00",
         "7/3/2014": "13:00:00",
         "7/3/2017": "13:00:00",
+        "7/3/2018": "13:00:00",
+        "7/3/2019": "13:00:00",
         "11/24/2006": "13:00:00",
```

Comment:
Should make a note in the calendar for Jan-2020 to update this before the next July 4th early close.